proto/store.proto

```diff
@@ -0,0 +1,115 @@
+syntax = "proto3";
+package pluginv2;
+
+option go_package = "./;pluginv2";
+
+message EntityReference {
+  string kind = 1;
+  string grn = 2; // includes folder pattern -- may include git branch info
+}
+
+//-----------------------------------------------
+// GET
+//-----------------------------------------------
+
+message GetRequest {
+  PluginContext pluginContext = 1;
+
+  EntityReference ref = 2;
+}
+
+message GetResponse {
+  int32 code = 1;
+  string grn = 2;
+  string kind = 3;  // dashboard | alert | datasource | panel (library) | ...  
+  string name = 4;
+  string description = 5;
+  string schemaVersion = 6; // [scumata version?]
```

Comment:
this is an interesting question of metadata vs. in-object data. Thema can work either way (https://github.com/grafana/thema/issues/16). In general it's _probably_ preferable to have the version in the body. But it could also always be included in metadata, as well. Or we could end up with something similar to a CRD framework for wrapping metadata around the objects themselves (in Go).

Reply:
intersting -- yes -- this is how things seem a bit better in https://github.com/grafana/grafana-plugin-sdk-go/pull/482